hudi-client/pom.xml

```diff
@@ -85,6 +85,11 @@
       <groupId>log4j</groupId>
       <artifactId>log4j</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.28</version>
```

Comment:
Since we will introduce the slf4j, IMO, we should extract the version of this dependency to the parent pom file and let other modules introduce the same version number. WDYT?

Reply:
OK, Thank you for your advice，I'll try it!